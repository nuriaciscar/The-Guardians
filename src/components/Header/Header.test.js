// import Header from ""./Header.js""
// describe("Given a component Header", {
//     describe("When it ")
// })
