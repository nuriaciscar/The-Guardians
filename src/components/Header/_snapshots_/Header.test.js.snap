exports[`Then it should render a header with a logo and his navigation`] = `
<div className="header__top-container">
        <NavLink to="/home" className="header-logo" exact>
          <img
            src={sourceLogo}
            className="header-logo__image"
            alt="logo of the guardian"
            height="100"
            width="290"
          />
        </NavLink>
        <div className="header-actions">
          <div className="header-user-area">
            <NavLink to="/mynews" className="header-user-area__my-news" exact>
              MY NEWS
            </NavLink>

            <button className="header-user-area__button header-user-area__button--sign-up">
              Sign up
            </button>

            <button className="header-user-area__button header-user-area__button--login">
              Login
            </button>
          </div>
          <div className="header-nav">
            <ul className="header-nav__list">
              <a href="/home#news" className="header-nav__list-element">
                NEWS
              </a>
              <a href="/home#sports" className="header-nav__list-element">
                SPORTS
              </a>
              <a href="/home#science" className="header-nav__list-element">
                SCIENCE
              </a>
              <a href="/home#lifestyle" className="header-nav__list-element">
                LIFESTYLE
              </a>
            </ul>
          </div>
        </div>
      </div>
      <div className="header-bottom-container">
        <div className="header-runner">
          <div className="header-runner__element">
            <p className="header-runner__element__text__color">Breaking News</p>
          </div>
          <div className="header-runner__element">
            <p className="header-runner__element__text">
              Mario is exploting his students
            </p>
          </div>
          <div className="header-runner__element">
            <p className="header-runner__element__text">...Again</p>
          </div>
        </div>
      </div>
`;